### DISCUSSION TITLE
I lost all collections after the update!

### DISCUSSION THREAD

**Discussion by Pavel61981** [thunderclient/thunder-client-support#851] [General]
The collections are in the thunderCollection.json file
Requests are in the thunderclient.json file
But they are not displayed in the client!

**Comment by rangav** [comment]
Hi @Pavel61981 

Please restart vscode, it should work.

where is your data saved? In workspace or default location?

**Reply by Pavel61981** [reply]
I restarted several times. It was empty. Even created a new collection. Now deleted the new collection and restarted again. My collections are back. Weird

**Reply by rangav** [reply]
where is your data saved? In workspace or default location?

**Reply by Pavel61981** [reply]
default location

**Reply by rangav** [reply]
Thanks for sharing.
